electron-app/src/server/websites/telegram/telegram.service.ts

```diff
@@ -482,26 +462,36 @@ export class Telegram extends Website {
 
     for (const channel of data.options.channels) {
       this.checkCancelled(cancellationToken);
-      const [channel_id, access_hash] = channel.split('-');
-      const peer = {
-        _: 'inputPeerChannel',
-        channel_id,
-        access_hash,
-      };
+
       response = await this.callApi(accountData.appId, 'messages.sendMessage', {
         random_id: Date.now(),
         message: description,
         entities: entities,
         silent: data.options.silent,
-        peer,
+        peer: this.getPeer(channel),
       });
 
-      await WaitUtil.wait(2000);
+      await WaitUtil.wait(1000);
```

Comment:
q: any reason in particular to reduce the wait? Their rate-limits are pretty aggressive when they trigger.

Reply:
yeah they are but they are a lot lower then 2 seconds. Actually i made this change because loading large images to telegram is slower then to any other site.